package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wdio-screenshot",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Can you please undo this? [np](https://github.com/sindresorhus/np) does this automatically when publishing a new release,